openmetadata-ui/src/main/resources/ui/src/components/Settings/Users/users.less

```diff
@@ -339,6 +347,13 @@
   background: #0950c5;
   position: absolute;
   bottom: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.persona-profile-edit-cancel.ant-btn-icon-only.ant-btn-sm svg {
+  margin-top: 0px !important;
```

Comment:
Not sure if we are trying to remove this margin from the icon ?
<img width="454" alt="image" src="https://github.com/user-attachments/assets/ca43081c-2339-4661-b5d3-96e1a9a2a96f" />

Reply:
yes , its this only. Have removed this style now.